ex03.ex

```diff
@@ -149,9 +154,39 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
-
-
+  def won( { x,  x,  x,
```

Comment:
## +2

Two bonus points. One for the 2d layout, and one for the _1, _2 naming.